go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/go-logr/logr v1.4.1
 	github.com/google/go-cmp v0.6.0
 	github.com/kubeflow/common v0.4.7
-	github.com/kubeflow/mpi-operator v0.4.0
```

Comment:
Uhm... this is not released.

Let me see if I can make a release.

Reply:
@alculquicondor Ohh that would be awesome! Thanks, let me know if that's possible

Reply:
Is there any dep that needs an upgrade in the mpi-operator repo?

Reply:
I think that we already used the latest version: https://github.com/onsi/ginkgo/releases/tag/v2.17.1
What does it mean?

Reply:
@tenzen-y that is correct. I meant that the `mpi-operator` repo has all their dependencies upgraded since their latest release `0.4.0`, except for `ginkgo` and `klog`, but I think those should be fine to keep as they are.

Reply:
I see. Thank you.
I understand that we should bump those libs in the mpi-operator...
But, it would take some time since those libs don't have backward compatibility...

Reply:
We (Aldo and me) are maintainers of the mpi-operator, and then we will release v0.5.0 this week (today or tomorrow). So, after released the new mpi-operator, we can update the mpi-operator in this repository as well.

Reply:
Awesome, thanks a lot @tenzen-y . I guess we can keep this PR opened until then, then I'll update to use `v0.5.0` here.

Reply:
SGTM

Reply:
@ChristianZaccaria I published a new mpi-operator v0.5.0. So, could you update this PR?

Reply:
@tenzen-y Thank you so much! I've just updated the PR.